proxy/src/graphql/schema.rs

```diff
@@ -295,13 +306,17 @@ impl Query {
     }
 
     fn user(ctx: &Context, handle: juniper::ID) -> Result<Option<juniper::ID>, error::Error> {
-        Ok(futures::executor::block_on(
+        let maybe_handle = futures::executor::block_on(
             ctx.registry
                 .read()
                 .expect("unable to acquire read lock")
                 .get_user(handle.to_string()),
-        )?
-        .map(juniper::ID::new))
```

Comment:
Why the change from `map(juniper::ID::new))` to `match`? It adds another few lines of code with no added value.